apps/src/templates/GameButtons.jsx

```diff
@@ -19,12 +21,30 @@ export const FinishButton = () => (
   </button>
 );
 
+const sendAnalyticsEvent = () => {
+  const isSignedOut =
+    document
+      .querySelector('script[data-issignedout="true"]')
+      ?.getAttribute('data-issignedout') === 'true';
```

Comment:
Curious about the necessity of this second line!

Reply:
Updated!